Gruntfile.js

```diff
@@ -38,26 +38,30 @@ module.exports = function(grunt) {
         configuration: grunt.file.readJSON("tslint.json")
       },
       src: [
-        "src/*.ts",
-        "src/formatters/**/*.ts",
-        "src/language/**/*.ts",
-        "src/rules/**/*.ts"
+        'src/*.ts',
```

Comment:
let's consistently use double quotes -- that's our standard everywhere else. feel free to file an issue and revert this file to what it was using originally.